app/main/routes.py

```diff
@@ -433,24 +441,26 @@ def search_transferring_body(_id: uuid.UUID):
         3: {"search_terms": ""},
     }
 
-    if query:
-        str_items = ""
+    if query.strip():
```

Comment:
since you now have `_get_search_terms`, I htink we should adjust that and reuse it here if necessary instead of having 2 lots of similar search term separation logic

Reply:
both the functionalities are different from each other, will have a look at how can I combine to gather

Reply:
yeah they are slightly different but seem similar enough to extract the similar parts